sql/moz-fx-data-marketing-prod/ga_derived/blogs_daily_summary_v1/query.sql

```diff
@@ -0,0 +1,44 @@
+SELECT
+  date,
+  deviceCategory,
+  operatingSystem,
+  browser,
+  LANGUAGE AS language,
+  country,
+  standardized_country_list.standardized_country AS standardized_country_name,
+  source,
+  medium,
+  campaign,
+  content,
+  blog,
+  subblog,
+  SUM(sessions) AS sessions,
+  SUM(downloads) AS downloads,
+  SUM(socialShare) AS socialShare,
+  SUM(newsletterSubscription) AS newsletterSubscription
+FROM
+  `moz-fx-data-marketing-prod.ga_derived.blogs_sessions_v1` AS sessions_table
+LEFT JOIN
+  `moz-fx-data-marketing-prod.ga_derived.blogs_goals_v1`
+USING
+  (date, visitIdentifier)
+LEFT JOIN
+  `moz-fx-data-marketing-prod.static.standardized_country_list` AS standardized_country_list
```

Comment:
Would be great if this standardized country list was put somewhere central to be shared across Moz. I'd made this list coz I was tired of trying to reconcile between GA and Telemetry. But I think I'm the only one using it / keeping it up to date. Would be great if we could make it something better maintained by Data Eng that applies against other data sources and is used centrally. *This is a nice to have outside this project*

Reply:
FYI I just have a local csv that I update when the need arises and the I overwrite that table. Maybe that csv can live somewhere else.

Reply:
We already have such a table. See `country_names` and `country_codes` in https://github.com/mozilla/bigquery-etl/tree/master/sql/moz-fx-data-shared-prod/static

The country_names table in particular includes listings for each of the country names that show up in the FxA metrics pipeline. If GA produces different logical country names, it likely makes sense to add them to `country_names` and use that int this query as well.

Reply:
I made it into a csv here:
https://github.com/mozilla/bigquery-etl/blob/07603860fdb71bcabcd8b05fcc9e12267abaab1d/sql/moz-fx-data-marketing-prod/static/standardized_country_list/data.csv

We can just move this into shared-prod.  The logic for this table is a little different from the existing ones so I think a new table works best.